WORKSPACE

```diff
@@ -86,3 +86,12 @@ tf_workspace1()
 load("@//tensorflow:workspace0.bzl", "tf_workspace0")
 
 tf_workspace0()
+
+# Register the platform and toolchain to support clang-cl on Windows
+register_execution_platforms(
```

Comment:
Toolchains should be registered down in https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace2.bzl#L94

Reply:
Done